core-java/src/main/java/com/baeldung/algorithms/annealing/City.java

```diff
@@ -2,21 +2,35 @@
 
 import lombok.Data;
 
-@Data
-public class City {
+@Data public class City {
```

Comment:
One note, is this class relevant to the scope of the article? Also, the formatting is a little bit off. I already addressed this in another comment